ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -358,6 +387,11 @@ export default defineComponent({
       router.push({ name: RouteNames.MHRSEARCH_CONFIRM })
     }
 
+    const hasSimilarOwners = (item: ManufacturedHomeSearchResultIF): boolean => {
```

Comment:
It's actually the same results, not similar. I would rename it to `hasMultipleSelections`.

Reply:
good call